pkg/cache/cache.go

```diff
@@ -163,6 +174,10 @@ func New(defaultAudience, prefix string, defaultRegionalSTS bool, defaultTokenEx
 						return
 					}
 				}
+				if _, ok := sa.Annotations[c.annotationPrefix+"/"+pkg.RoleARNAnnotation]; ok {
```

Comment:
the indenting looks very weird here, is it tabs vs spaces? gofmt?

Reply:
also is it intentional to call Dec twice?

Reply:
changed the approach.